cmd/execute.go

```diff
@@ -96,7 +98,11 @@ func executeTransformers() {
 	// Use WaitGroup to wait on both goroutines
 	var wg sync.WaitGroup
 	if len(ethEventInitializers) > 0 {
-		extractor := logs.NewLogExtractor(&db, blockChain)
+		repo, repoErr := repositories.NewCheckedHeadersRepository(&db, genConfig.Schema)
+		if repoErr != nil {
+			LogWithCommand.Fatalf(fmt.Errorf("failed to create checked headers repository: %w", repoErr).Error())
```

Comment:
nbd but if we're just logging the error and not returning it, we could probably simplify this as:
```
LogWithCommand.Fatalf("failed to create checked headers repository: %s", repoErr.Error())
```

Reply:
👍 - you have me trained to put fmt.Errorf everywhere!